setup.py

```diff
@@ -122,13 +122,7 @@ def write_version_py(filename=None):
         "Programming Language :: Python :: 3.5",
         "Topic :: Scientific/Engineering",
     ],
-    install_requires=[
-        "scipy >= 0.13",
-        "numpy >= 1.8",
-        "netCDF4 >= 1.0.6",
-        "matplotlib >= 1.3.1",
-        "pandas >= 1.0",
-    ],
+    install_requires=[req.strip() for req in open("requirements.txt")],
```

Comment:
We're trying to move away from using `setup.py` in favour of `pyproject.toml`. A precondition of that is not having dynamic content in `setup.py` (ask @rod-glover about his most recent work in `pycds`). As such, it would be better to not add any dynamic content here.